src/components/registerForm.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div class="mx-auto p-5 rounded-3xl w-min" style="background-color: #313030;">
+        <div class="p-5">
+            <h1>Register</h1>
+            <p>Fill all fields</p>
+            <div class="mt-12">
+                <form>
+                    <p v-if="parsedMessage" class="text-red-600 w-full mb-4 text-xs">{{ parsedMessage }}</p>
+                    <div class="input-div">
+                        <input class="w-full darkcolor lightbg mb-6 p-1.5 rounded-xl" type="text" placeholder="Name" v-model="name" />
```

Comment:
Create a BaseInput component to use instead of input

Reply:
Changes added